source/material_model/reactive_fluid_transport.cc

```diff
@@ -71,6 +67,11 @@ namespace aspect
       // Td polynomials are defined in equations 15, B3, B11, and B19.
       for (unsigned int i = 0; i<devolatilization_enthalpy_changes.size(); ++i)
         {
+          // Pressure, which must be in GPa for the parametrization, or GPa^-1. The polynomials for each lithology
+          // breaks down above certain pressures, make sure that we cap the pressure just before this break down.
+          // Also if the pressure is too small, make is 1e-12 to avoid a division by 0.
+          const double pressure = std::min(std::max(1e-12, in.pressure[q]/1.e9), pressure_cutoffs[i]);
+          const double inverse_pressure = 1.0/pressure;
```

Comment:
```suggestion
          // Introduce minimum pressure to avoid a division by 0.
          const double minimum_pressure = 1e-12;
          const double pressure = std::min(std::max(minimum_pressure, in.pressure[q]/1.e9), pressure_cutoffs[i]);
          const double inverse_pressure = 1.0/pressure;
```